tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/MinorDottySuite.scala

```diff
@@ -577,4 +577,118 @@ class MinorDottySuite extends BaseDottySuite {
       )
     )
   }
+
+  test("simple-derives") {
+    val derivesEnum = """|enum Tree[T] derives Eq, Ordering, Show {
+                         |  case Branch
+                         |  case Leaf
```

Comment:
I see case Branch/Leaf are from example, but maybe we can omit them here to have shorter test. Also can stay, just loose proposal :)

Reply:
it doesn't add that much, I already reduced the example to managable size I think